### PR TITLE
Feature/geography feedback improvements

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Update tooltip styling and allow React Nodes to be passed in rather that just strings
- Ensure documents in list always render the same
- Reserve space where 'federal' flag appears

### CODE REVIEW

File: frontend/src/components/blocks/CountryHeader.tsx [hunk 1]
```diff
@@ -49,7 +49,18 @@ export const CountryHeader = ({ country }: TProps) => {
...
-                    tooltip="The annually published Global Climate Risk Index analyses to what extent countries have been affected by the impacts of weather-related loss events (storms, floods, heat waves etc.)."
+                    tooltip={
+                      <>
+                        <p className="mb-4">
+                          The annually published Global Climate Risk Index analyses to what extent countries have been affected by the impacts of weather-related loss events
+                          (storms, floods, heat waves etc.).
+                        </p>{" "}
+                        Published by German Watch{" "}
+                        <a href="" target="_blank" className="underline">
+                          https://www.germanwatch.org/en/cri
```
Comment (joel-wright) [inline frontend/src/components/blocks/CountryHeader.tsx:60]: this is going to be fun when we're providing multiple UI languages...

**Review by joel-wright — APPROVED** [review]
LGTM

File: frontend/src/components/blocks/CountryHeader.tsx (hunk 1 above)
Comment (PatFawbertMills) [reply frontend/src/components/blocks/CountryHeader.tsx:60]: Yep there's gonna be a fun little refactor to come...

**Comment by PatFawbertMills** [comment]
Tooltip refactor to accept react nodes and some small layout updates to the geography information header

**Pull request merged**